src/languages/en-US.json

```diff
@@ -1,4 +1,353 @@
+	"lockdown_unlocked": "{{emote_success}} Successfully unlocked channel <#{{channel}}>",
+
+	"mute_muterole_not_found": "{{emote_warn}} This server does not have a mute role configured!\n{{emote_wrench}} **Tip:** You can configure the mute role using the `!configure mute_role` command",
+	"mute_not_muted": "{{emote_warn}} **{{target.tag}}** `({{target.id}})` is not currently muted!",
+	"mute_rejoin": "{{emote_warn}} **{{user.tag}}** `({{user.id}})` has rejoined the server before their mute expired and has been automatically muted again.",
+	"mute_already_muted": "{{emote_warn}} **{{target.tag}}** `({{target.id}})` is already muted!",
+	"unmute_confirm": "{{emote_warn}} **{{target.tag}}** `({{target.id}})` has not been muted by Bulbbot. Are you sure you want to unmute them?\n*Unmuting a user not muted by Bulbbot will only remove the muted role from them*",
+	"unmute_special": "{{emote_success}} **{{target.tag}}** `({{target.id}})` has been unmuted for **{{reason}}** ",
+
+	"infraction_not_found": "{{emote_fail}} Infraction **#{{infraction_id}}** could not be found!",
+	"infraction_delete_confirm": "{{emote_warn}} Are you sure you want to delete infraction **#{{infraction_id}}**? This action __**cannot**__ be undone.\n**Offender:** {{target.tag}} `({{target.id}})`\n**Moderator:** {{moderator.tag}} `({{moderator.id}})`\n**Reason:** {{reason}}",
+	"infraction_delete_success": "{{emote_success}} Infraction **#{{infraction_id}}** has been successfully deleted!",
+	"infraction_info_inf_id": "**Infraction ID:** {{infraction_id}}\n",
+	"infraction_info_target": "**Target:** {{target.tag}} `({{target.id}})`\n",
+	"infraction_info_moderator": "**Moderator:** {{moderator.tag}} `({{moderator.id}})`\n",
+	"infraction_info_created": "**Created:** {{created}}\n",
+	"infraction_info_active": "**Active:** {{active}}",
+	"infraction_info_expires": "**Expires:** {{expires}}",
+	"infraction_info_reason": "\n\n**Reason:** {{reason}}",
+	"infraction_search_not_found": "{{emote_warn}} **{{target.tag}}** `({{target.id}})` has no infractions!",
+	"infraction_list_not_found": "{{emote_warn}} This server does not seem to have any infractions!",
+	"infraction_claim_success": "{{emote_success}} You have claimed responsibility over infraction **#{{infraction_id}}**",
+	"infraction_update_success": "{{emote_success}} Infraction **#{{infraction_id}}** has been updated!",
+
+	"infraction_interaction_reply": "{{emote_warn}} **{{target.tag}}'s** `({{target.id}})` infractions",
+	"infraction_interaction_placeholder": "No Infractions Selected",
+	"infraction_interaction_description": "Click to view more info",
+
+	"commands_help": "You can find all the commands Bulbbot offers over **[here](https://docs.bulbbot.mrphilip.xyz/command-list)**",
+
+	"help_unable_to_find_command": "`{{commandName}}` is not a valid command, for a full list of commands visit our website <https://docs.bulbbot.mrphilip.xyz/command-list>"
+=======
```

Comment:
Everything below this line except the closing `}` needs to be removed else it will break